### PR TITLE
*: add label distribution metrics

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
add label distribution metrics.

### What is changed and how it works?
more convenient to get the distribution information of the cluster

### Check List 

Tests 

 - Unit test

Related changes
 - Need to update the `tidb-ansible` repository

### CODE REVIEW

File: server/store_statistics.go [hunk 1]
```diff
@@ -14,13 +14,19 @@
...
+	"fmt"
 	"strconv"
 
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/pd/server/core"
 	"github.com/pingcap/pd/server/namespace"
 )
 
+const (
+	unknow    = "unknow"
```
Comment (rleungx) [inline server/store_statistics.go]: How about `unknown`?

**Comment by huachaohuang** [comment]
Why does the Jenkins unit test always fail?

**Comment by rleungx** [comment]
@huachaohuang `jenkins-ci-pd/unit-test` is our old CI, which is not stable. And this week, I spent some time to make PD's test run in parallel in `idc-jenkins-ci/build`. You can see the test result on [here](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/pd_test/detail/pd_test/108/pipeline). But for now it will run these two CI at the same time. So the old CI could be failed sometimes. Maybe we can switch to `idc-jenkins-ci/build` which is more stable in the next week.

**Comment by huachaohuang** [comment]
OK, thanks.

**Pull request merged**